apps/core/src/hooks/useTransactionSummary.ts

```diff
@@ -60,6 +60,7 @@ export function useTransactionSummary({
 				objectSummary,
 				status: transaction.effects?.status.status,
 				timestamp: transaction.timestampMs,
+				upgradedSystemPackages: transaction.effects?.mutated,
```

Comment:
This seems like kind of a misnomer? This is just all mutated objects right?

Reply:
I am filtering for owner `Immutable`

Reply:
Can you filter here vs in the UI?